Adds documentation for the Data Prepper geoip processor and geoip_service extension (#6772)
opensearch-project/documentation-website: _data-prepper/managing-data-prepper/extensions/extensions.md

---
layout: default
title: Extensions
parent: Managing Data Prepper
has_children: true
nav_order: 18
---

# Extensions

> Data Prepper extensions Data Prepper functionality outside of core Data Prepper pipeline components.

Reviewer: ```suggestion
Data Prepper extensions provide Data Prepper functionality outside of core Data Prepper pipeline components.
```

Many extensions provide configuration options that give Data Prepper administrators greater flexibility over Data Prepper's functionality.

> Extension configurations are available to configure in the `data-prepper-config.yaml` file under the `extensions:` YAML block.

Reviewer: ```suggestion
Extension configurations can be configured in the `data-prepper-config.yaml` file under the `extensions:` YAML block.
```
